xml/admin_backup.xml

```diff
@@ -0,0 +1,394 @@
+  <para>
+   In some failure scenarios (network disconnect, simple hardware failure) nodes
+   can be recovered without additional action. Simply bringing the machines back
+   online will suffice to return them to the cluster.
+  </para>
+  <para>
+   In case of severe hardware failure (loss of hard drive, data corruption),
+   generally speaking, a replacement is advised and nodes should be replaced
+   with fresh (virtual) machines.
+  </para>
+ </note>
+
+  <sect2 xml:id="sec.admin.backup.admin">
+   <title>Admin Node Failure</title>
+   <para>
+    We assume:
+   </para>
+
+    <itemizedlist>
+     <listitem>
+      <para>
+       The &admin_node; has failed and is unrecoverable
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The rest of the cluster is unaffected
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The IP address / hostname for the replacement node will be identical
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       A backup of the &admin_node; has been taken from a healthy cluster state
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The backup procedure is performed regularly
+      </para>
+     </listitem>
+    </itemizedlist>
+
+   <para>
+    The <xref linkend="sec.admin.backup.admin.restore"/> will involve deploying
+    a new &admin_node; machine and restoring the previous data onto it.
+    &productname; will then resume management of the cluster. Existing workloads
+    should not be affected.
+   </para>
+   <procedure>
+    <title>Backup procedure</title>
+    <step>
+     <para>
+      SSH into the &admin_node;.
+     </para>
+    </step>
+    <step>
+     <para>
+      Change to the <filename>/var/tmp</filename> directory and package
+      important data into a tar file archive:
+     </para>
+<screen>&prompt.root;<command>
+tar czf <replaceable>backup-admin-node-$( date +'%Y-%m-%d' ).tgz</replaceable> -C / \
+    var/lib/misc/salt-master-credentials \
+    usr/share/salt \
+    etc/pki usr/share/caasp-container-manifests \
+    etc/salt/pki \
+    etc/caasp \
+    var/lib/misc/salt/cache \
+    etc/caasp/haproxy \
+    var/lib/mysql \
+    var/lib/misc/infra-secrets \
+    var/lib/misc/ldap \
+    var/lib/misc/ldap-config \
+    var/lib/misc/ssh-public-key \
+    var/lib/misc/infra-secrets \
+    etc/salt/pillar \
+    var/lib/misc/velum-secrets</command>
+ </screen>
+    </step>
+    <step>
+     <para>
+      Then, export the MariaDB database and archive it:
+     </para>
+ <screen>&prompt.root;<command>docker exec $(docker ps -qf name="velum-mariadb") mysqldump \
+ --password=$(cat /var/lib/misc/infra-secrets/mariadb-root-password) \
+ velum_production | gzip > <replaceable>/var/tmp/db-backup-[$(date+'%Y-%m-%d')].sql.gz</replaceable></command>
+ </screen>
+    </step>
+    <step>
+     <para>
+      Finally, copy the created files <filename>backup-admin-node-[date].tgz</filename>
```

Comment:
The `tar` command for backup-admin-node-.tgz doesn't produce `-[2019-02-26].tgz` but `-2019-02-26.tgz` without square brackets.